src/app/pages/LiquidityMining/components/AddLiquidityDialog/AddLiquidityDialogV1.tsx

```diff
@@ -170,6 +171,7 @@ export const AddLiquidityDialogV1: React.FC<IAddLiquidityDialogV1Props> = ({
                 tx.loading || !valid || !canInteract || addliquidityLocked
               }
               className="tw-rounded-lg"
+              data-action-id="yieldFarm-deposit-button"
```

Comment:
```suggestion
              data-action-id="yieldFarm-liquidityModal-confirm"
```

Spec has been updated to use a shared attribute name for the confirm buttons on both add/remove liquidity modals